SPECS/perftest/perftest.spec

```diff
@@ -0,0 +1,80 @@
+Summary:        IB Performance tests
+Name:           perftest
+# Update extended_release with version updates
+Version:        4.5
+Release:        1%{?dist}
+License:        BSD 3-Clause, GPLv2
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Productivity/Networking/Diagnostic
+URL:            https://www.openfabrics.org
+Source0:        https://linux.mellanox.com/public/repo/doca/1.3.0/extras/mlnx_ofed/5.6-1.0.3.3/SOURCES/perftest_4.5.orig.tar.gz#/%{name}-%{version}.tar.gz
+%global         extended_release 0.14.gd962d8c.56068 
+BuildRequires:  libibumad-devel
+BuildRequires:  libibverbs-devel
+BuildRequires:  librdmacm-devel
+BuildRequires:  pciutils-devel
+
+%description
+gen3 uverbs microbenchmarks release: %extended_release
+
+%prep
+%autosetup -p1
+
+%build
+%configure \
+%{__make}
```

Comment:
This looks kinda wack to me, so just making sure- the configure script wants to be passed the path to make?

Reply:
Yup this is a typo which didn't raise a failure. Fixing and testing